src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml

```diff
@@ -130,7 +130,7 @@
                                       Style="{StaticResource InCanvasScrollViewerStyle}"
                                       CanContentScroll="True"
                                       Width="250">
-                            <ContentControl Template="{StaticResource MemberControlTemplate}" />
+                            <ContentControl Template="{StaticResource InCanvasMemberControlTemplate}" />
```

Comment:
I discussed with Zach and we are ok with library, browse, search, and in-canvas search showing just the method name and not the full signature.  The original intent was to just take the full signature off for in-canvas search, but we think it is ok to take it off in both places. This information is more legible in the tool tip.